domains/solomon.json

```diff
@@ -7,11 +7,11 @@
   },
 
   "target": {
-    "CNAME": {
-      "name": "solomon", 
-      "value": "solomonsweb.github.io"
+    "A": {
+      "name": "solomon.is-a-good.dev", 
```

Comment:
You don't need the is-a-good.dev on line 11.

Reply:
> > You don't need the is-a-good.dev on line 11.
> 
> according to my redirecting company, i do.

If you do decide to recreate a PR, please do not add the is-a-good.dev part. Doing this will have your record created as
solomon.is-a-good.dev.is-a-good.dev which most likely was not your intention.

Our format is used for our own systems, and what your hosting company might provide does not always match.